eng/Packages.Data.props

```diff
@@ -128,6 +128,7 @@
     <PackageReference Update="Azure.Storage.Blobs" Version="12.21.1" />
     <PackageReference Update="Azure.Storage.Queues" Version="12.19.1" />
     <PackageReference Update="Azure.Storage.Files.Shares" Version="12.19.1" />
+    <PackageReference Update="Azure.AI.Inference" Version="1.0.0-beta.2" />    
     <PackageReference Update="Azure.AI.OpenAI" Version="2.0.0" />
```

Comment:
This seems unrelated to the new SDK. Is this intentional?

Reply:
Yes, we have a common file where we add package references.

For Azure AI Projects, we take the dependency on [inference](https://github.com/Azure/azure-sdk-for-net/blob/feature/azure-ai-sdk/agents/sdk/ai/Azure.AI.Projects/src/Azure.AI.Projects.csproj#L18) for connection APIs.